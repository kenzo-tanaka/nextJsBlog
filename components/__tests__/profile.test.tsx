import React from "react";
import { createRenderer } from "react-test-renderer/shallow";
import Profile from "../profile";

const renderer = createRenderer();
test("profile", () => {
  renderer.render(<Profile />);
  const renderedOutput = renderer.getRenderOutput();
  expect(renderedOutput).toMatchSnapshot();
});
