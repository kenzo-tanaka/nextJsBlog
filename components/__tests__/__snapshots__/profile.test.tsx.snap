// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile 1`] = `
<div
  className="flex flex-col	items-center"
>
  <picture>
    <source
      srcSet="/images/profile.webp"
    />
    <Image
      alt="Picture of the author"
      className="rounded-full"
      height={100}
      src="/images/profile.webp"
      width={100}
    />
  </picture>
  <p
    className="text-gray-500 mt-4"
  >
    Personal tech blog /
     
    <Link
      href="/about"
    >
      <a
        className="underline"
      >
        resume
      </a>
    </Link>
  </p>
</div>
`;
